lib/disc.rb

```diff
@@ -110,10 +119,14 @@ def disc_options
       end
 
       def queue
-        disc_options.fetch(:queue, 'default')
+        @queue || Disc.default_queue
```

Comment:
BTW, not saying we *need* to do it this way. This is just another option for @pote to consider.